Package.swift

```diff
@@ -0,0 +1,8 @@
+import PackageDescription
+
+let package = Package(
+    name: "SQLiteSDK",
+    providers: [
+        .Apt("libsqlite3-dev")
```

Comment:
It's actually needed only on Linux (for apt-get). It prints a hint if sqlite is not installed.
3.8.2 on Ubuntu 14.04
3.11.0 on 16.04
I don't know which version is preinstalled on OS X.